keras/applications/imagenet_utils.py

```diff
@@ -58,6 +53,78 @@ def preprocess_input(x, data_format=None, mode='caffe'):
     return x
 
 
+def _preprocess_symbolic_input(x, data_format, mode):
+    global _IMAGENET_MEAN_CHW
+    global _IMAGENET_MEAN_NCHW
+    global _IMAGENET_MEAN_HWC
+    global _IMAGENET_MEAN_NHWC
+
+    if data_format is None:
+        data_format = K.image_data_format()
+    assert data_format in {'channels_last', 'channels_first'}
+
+    if mode == 'tf':
+        x /= 255.
+        x -= 0.5
+        x *= 2.
+        return x
+
+    imagenet_mean = [103.939, 116.779, 123.68]
+    if data_format == 'channels_first':
+        if K.ndim(x) == 3:
+            # 'RGB'->'BGR'
+            x = x[::-1, ...]
+            # Zero-center by mean pixel
+            if _IMAGENET_MEAN_CHW is None:
+                _IMAGENET_MEAN_CHW = np.array(imagenet_mean).reshape((3, 1, 1))
+            x -= _IMAGENET_MEAN_CHW.astype(np.float64)
```

Comment:
I missed this one during find and replacement.

By the way, what are your thoughts on @ozabluda's early comment:
> Maybe the global constant should be of type np.float64 then? Otherwise, if K.floatx() changes, that thread-safety issue you mentioned will arise. Or float32 constant will be wrong for float64 x, either because floatx() changed or because x was born this way. Then, maybe x -= ... can always (?) just take np.float64 right-hand-side? That's actually how it was before this PR. BTW, ILSVRC (nothing to say about Imagenet) has enough pixels to determine mean with accuracy for which float32 it's insufficient.